javatests/google/registry/export/ExportConstantsTest.java

```diff
@@ -78,16 +91,40 @@ public void testBackupKinds_matchGoldenBackupKindsFile() throws Exception {
   }
 
   @Test
-  public void testReportingKinds_areSubsetOfBackupKinds() throws Exception {
-    assertThat(ExportConstants.getBackupKinds()).containsAllIn(ExportConstants.getReportingKinds());
+  public void testReportingKinds_matchGoldenReportingKindsFile() throws Exception {
+    URL goldenReportingKindsResource =
+        getResource(ExportConstantsTest.class, GOLDEN_REPORTING_KINDS_FILENAME);
+    final Pattern stripComments = Pattern.compile("\\s*#.*$");
```

Comment:
This isn't really done... the helper function you added is just for printing out the update instructions, which is fine, but it doesn't address the duplication of the comment-stripping which is what I had described here.